src/pelib/ImageLoader.cpp

```diff
@@ -1075,29 +1074,14 @@ int PeLib::ImageLoader::Save(
 		return fileError;
 
 	// Write zeros to the rest of the file, up to size of image
-	if(saveHeadersOnly == false)
+	if((saveFlags & FileFlags::HeadersOnly) == 0)
```

Comment:
Is this equivalent to?
```cpp
if(!(saveFlags & FileFlags::HeadersOnly))
```
Wouldn't it be a little bit better with `!`?

Reply:
Solved in https://github.com/avast/retdec/pull/1029/commits/228bacec87f99312653a05024ff4c6102f591ad9